libtcmu_config.c

```diff
@@ -358,7 +358,8 @@ static void tcmu_parse_options(struct tcmu_config *cfg, char *buf, int len)
 
 int tcmu_load_config(struct tcmu_config *cfg, const char *path)
 {
-	char buf[TCMU_MAX_CFG_FILE_SIZE];
+	char *buf = malloc(TCMU_MAX_CFG_FILE_SIZE);
```

Comment:
This change is ok with me. What do you recommend for the limit when to switch to dynamic allocation should be?